README.md

```diff
@@ -67,5 +70,7 @@ This charm by default uses the latest version of the
 
 ## Contributing
```

Comment:
The contributing section is not needed, we have a `CONTRIBUTING.md` file now and that is easily enough to discover in GitHub, while this content is not needed IMO in CharmHub.

Reply:
Removed.